api-reference/10 UI Widgets/dxDiagram/1 Configuration/nodes/imageUrlExpr.md

```diff
@@ -5,7 +5,7 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specifies the name of a data source field or an expression that provides a node image's URL.
+Specifies the name of a data source field or an expression that provides a node image's URL or a node's image encoded to Base64.
```

Comment:
- a node image's & a node's image?
- "encoded to Base64" How can you encode something 'to' something else?